### DISCUSSION TITLE
Thermal runaway

### DISCUSSION THREAD

**Discussion by Giorvi** [Jyers/Marlin#771] [Q&A] [answered]
I’m wanting to run some longer prints on my Ender and I was wondering if there some sort of thermal runaway feature in the firmware? 

Thanks in advance.

**Comment by Timbo200** [comment, marked as answer]
Yes it's enabled in this firmware.

 I can also confirm it works as I was changing a nozzle and the heater was on and the fan removed.  I got distracted and 5 minutes later it activated and the buzzer went off.
